src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1246,14 +1246,16 @@ static bool addTriggeredSubscriptions_noCache
 
       LM_T(LmtMongo, ("adding subscription: '%s'", sub.toString().c_str()));
 
-      long long throttling       = sub.hasField(CSUB_THROTTLING)?       getIntOrLongFieldAsLongF(sub, CSUB_THROTTLING)       : -1;
-      long long lastNotification = sub.hasField(CSUB_LASTNOTIFICATION)? getIntOrLongFieldAsLongF(sub, CSUB_LASTNOTIFICATION) : -1;
+      long long           throttling         = sub.hasField(CSUB_THROTTLING)?       getIntOrLongFieldAsLongF(sub, CSUB_THROTTLING)       : -1;
+      long long           lastNotification   = sub.hasField(CSUB_LASTNOTIFICATION)? getIntOrLongFieldAsLongF(sub, CSUB_LASTNOTIFICATION) : -1;
+      std::string         notifyFormatString = sub.hasField(CSUB_FORMAT)? getStringFieldF(sub, CSUB_FORMAT) : "";
+      NotificationFormat  notifyFormat       = stringToNotificationFormat(notifyFormatString);
```

Comment:
Alternative:

```
NotificationFormat  notifyFormat       = stringToNotificationFormat(sub.hasField(CSUB_FORMAT)? getStringFieldF(sub, CSUB_FORMAT) : "");
```

Pros: you save a variable. More compact.
Const: longer and maybe more obscure.